index.js

```diff
@@ -231,3 +237,19 @@ selectTag.onchange = function (ev) {
     let tag = ev.target.selectedOptions[0].value;
     window.location.href = 'https://rawcdn.githack.com/lapo-luchini/asn1js/' + tag + '/index.html';
 };
+
+// zoom fix to have straight lines in treeview
```

Comment:
Is it possible to do this using CSS rules instead of custom JS?
Because I fear that direct style rules such as this would be stopped by CSP on official website.

Reply:
I added a version without JS. Looks fine on my screen.